.github/workflows/generate-pot.yml

```diff
@@ -2,6 +2,9 @@ name: Generate POT
 
 on:
   push:
+    branches:
+      - main
+      - translations
```

Comment:
This will also run on forks when they sync `aspirepress/aspireupdate:main` to `user/aspireupdate:main`, creating an additional commit every time (and subsequently showing that commit in any PRs).

Should the `WP_POT_GENERATOR` have an `if` to check the repository like the Playground Blueprint workflow?

```yaml
if: github.repository == 'aspirepress/AspireUpdate'
```

Reply:
I see for the other GA.

Reply:
Updated.